### DISCUSSION TITLE
Client users' signing sessions unreachable

### DISCUSSION THREAD

**Discussion by agrgal** [epoptes/epoptes#205] [Q&A]
This is an issue that gets me crazy. Some years ago, epoptes server detected clients before signing in their computers. As a teacher, I could control those session signing processes.

Any solution to this issue? I need to connect the server by means of local hostname (SERVER=servername.local). Server system is Ubuntu 22.04.3 LTS and clients Ubuntu 20.04.6 LTS (xubuntu)-

Thanks in advance.

**Comment by alkisg** [comment]
Go to a client. Press Ctrl+Alt+F3 to switch to vt3. Login as e.g. teacher there. Then type:

```
sudo -i
systemctl status epoptes-client
systemctl start epoptes-client
# At that point check if you see the client on the server epoptes
epoptes-client
# And again check if you see the client on the server epoptes
```

Then paste the whole terminal output here e.g. via photo.

**Reply by agrgal** [reply]
Thanks alkisg, 

Attaching client's output. Blank screen on server side although connected (under vt3), no matter before epoptes-client or after.

![20230908_131151](https://github.com/epoptes/epoptes/assets/12474907/e1c7e4c8-cea7-4743-a477-3f9af52057cf)

**Comment by alkisg** [comment]
I think you just have an old epoptes-client version that doesn't wait for MDNS.
Use the Epoptes PPA or the Epoptes proposed PPA to update the clients to the last version.
https://epoptes.org/documentation/ppa/

**Reply by agrgal** [reply]
Thanks. I'll be returning to IT class next monday and I'll try. Then, I'll report it.

**Reply by agrgal** [reply]
Hi, @alkisg. Me again.

it didn't work. 

- I added stable epoptes ppa to both server and client.
- I upgraded epoptes on server, but not epoptes-client. Suppose I don't need it.
- I upgraded epoptes & epoptes-client on client side. 
- I fetched certificate from server again, successfully.

I got same behaviour. Attaching new screenshot. 

![20230911_090141](https://github.com/epoptes/epoptes/assets/12474907/ca3ea020-6f1c-4490-a572-15972030b1f7)

**Reply by agrgal** [reply]
The maximun I can get (sometines, not always) is an STD thumbnail instead of MAC thumbnail. 
![Captura de pantalla -2023-09-11 09-16-15](https://github.com/epoptes/epoptes/assets/12474907/b05fb22f-6aef-44fe-b794-97f785b5e85e)

**Comment by alkisg** [comment]
@agrgal if you want you can come in the chat room for live troubleshooting, it'll be faster.

https://ltsp.org/guides/chat-room/

**Reply by agrgal** [reply]
Thanks alkisg, but not now. This is one of a bucket of things I have to do. Sorry.

**Comment by alkisg** [comment]
Yeah I too have like a million things to do today, that's why I preferred to solve this quickly.
Let's talk in the chat room whenever both of us are available.
